plugins/catalog-backend/examples/locationref.yaml

```diff
@@ -0,0 +1,8 @@
+---
+apiVersion: backstage.io/v1beta1
+kind: LocationRef
+metadata:
+  name: location-1
+spec:
+  type: github
```

Comment:
Random thought here, what do you think about moving this to top-level, kinda like k8s secrets? https://kubernetes.io/docs/concepts/configuration/secret/

Not sure I like the idea, but it would make the schema selection a bit more natural, as in depending on the `type` you get different schemas for `spec`

Reply:
Hm maybe? Will have to change the root entity envelope definition a bit then...